src/realm/sync/client.hpp

```diff
@@ -350,6 +350,9 @@ class Session {
         bool simulate_integration_error = false;
 
         std::function<SyncClientHookAction(const SyncClientHookData&)> on_sync_client_event_hook;
+
+        /// Schema version
+        uint64_t schema_version = -1; // = ObjectStore::NotVersioned
```

Comment:
Recommend updating the comment to indicate it's for FLX only, or update the value to `flx_schema_version` since this feature is only for FLX sync.